python/grass/temporal/core.py

```diff
@@ -989,22 +966,19 @@ def upgrade_temporal_database(dbif):
         dbif.close()
         return
 
-    template_path = get_sql_template_path()
+    template_path = Path(get_sql_template_path())
     try:
-        upgrade_db_sql = open(
-            os.path.join(
-                template_path,
-                "upgrade_db_%s_to_%s.sql" % (upgrade_db_from, tgis_db_version),
-            ),
-            "r",
-        ).read()
+        upgrade_db_sql = (
+            template_path
+            / "upgrade_db_{}_to_{}.sql".format(upgrade_db_from, tgis_db_version)
```

Comment:
This will trigger Pylint warning to simplify with f-string, no?

Reply:
They weren't for now... Nothing got triggered. I changed it to have a diff more closely matching the lines. Otherwise, I would have needed brackets for the percent modulo, and it wrapped on more lines.

When it will be time to enforce f-strings, there's thousands of places to change anyways...